src/Command/Docker/Build.php

```diff
@@ -199,6 +220,8 @@ public function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
+        $config->set(DeveloperCompose::SYNC_ENGINE, $syncEngine);
```

Comment:
`sync-engine` is already should be set on line 204, why do you need to set it again?